packages/discord.js/typings/index.d.ts

```diff
@@ -6363,6 +6365,12 @@ export interface StageInstanceEditOptions {
   privacyLevel?: StageInstancePrivacyLevel;
 }
 
+export interface SupportingInteractionResolvedData {
+  client: Client;
+  guild?: Guild;
+  channel?: Channel;
```

Comment:
it needs to be a channel with messages in it

Reply:
Done with de41e9540f9cead1708dd1ed2aab741a6a132aef.